tags

```diff
@@ -0,0 +1,5552 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
What's the usage of this file?

Reply:
Oops.
It's the tag file for the vim editor.
I Remove it.